webapp/src/ts/app.component.ts

```diff
@@ -110,14 +113,21 @@ export class AppComponent {
     this.globalActions = new GlobalActions(store);
 
     moment.locale(['en']);
+    this.languageService.set()
```

Comment:
Please put `set` on a new line.  https://docs.communityhealthtoolkit.org/contribute/code/style-guide/#method-chaining

Applies across the PR.

Reply:
Done.